src/handler/src/Command.ts

```diff
@@ -9,7 +9,13 @@ export default interface Command {
 	description?: string;
 	clientPermissions?: string[];
 	userPermissions?: string[];
-	execute(message: Message, args: Args, locale: string): unknown | Promise<unknown>;
+	regExp?: RegExp;
+	execute(message: Message, args: Args, locale: string, executionContext: ExecutionContext): unknown | Promise<unknown>;
+}
+
+export enum ExecutionContext {
+	PREFIXED = 0,
```

Comment:
Any specific reason for the init with 0?

Reply:
Really just verbosity as per <https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums> numeric and 0-indexed is the default anyways, if you feel it's cleaner without the init i'll of course gladly remove it or change it to the respective "flag" string, if that's preferred

Reply:
Yeah, we don't need that. If its 0-indexed we can just remove the initialization.

Reply:
👍  done in https://github.com/Naval-Base/yuudachi/pull/775/commits/3a90be1a65701a33051bc176fa6ec68d328b6cde